examples/axes_grid1/demo_axes_divider.py

```diff
@@ -3,6 +3,7 @@
 Demo Axes Divider
 =================
 
+Example using Axes divider to calculate location of axes and create a divider for them using exisiting axes instances.
```

Comment:
This line isn't pep8: it's too long.

You can probably configure your text editor to show you the lines that don't follow pep8 automatically.